tests/test_flatAndIlluminationCorrection.py

```diff
@@ -98,6 +98,30 @@ def testIllum2(self):
     def testIllum3(self):
         self.doIllum(scaling=3.7)
 
+    def testGainAndReadnoise(self):
+        import lsst.afw.image as afwImage
+        from lsst.afw.cameraGeom.testUtils import DetectorWrapper
+        from lsst.ip.isr import IsrTask
```

Comment:
Why are there imports inside the test method?

Reply:
There's no downside, but I'm happy to move them.  That's probably the rule anyway (I should know)

Reply:
Yes. As-is, this will fail flake8 (which you apparently aren't running?).

Reply:
Imports have now been removed from the method and there are no flake8 violations